### ISSUE TITLE
Low AP and many localization errors when evaluating with higher input resolution

### ISSUE THREAD

**Issue by netanel-s** [tensorflow/models#5157]
### System information
- **What is the top-level directory of the model you are using**: object_detection
- **Have I written custom code**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: Tried originally with 1.8.0, but then also with 1.10.
- **Bazel version (if compiling from source)**: -
- **CUDA/cuDNN version**: CUDA 9
- **GPU model and memory**: 1080Ti, 12GB
- **Exact command to reproduce**: evaluating pretrained ssdlite_mobilenet_v2_coco_2018_05_09 using object_detection/model_main.py when changing configuration of the image_resizer.

### Describe the problem
 I tried increasing input resolution hoping to get better AP. I tried changing image_resizer to both:
fixed_shape_resizer {heigh :480, width: 640}, and 
keep_aspect_ratio_resizer {min_dimension: 480, max_dimension: 640}.
Instead, I got lower AP and many localization errors in both cases.
I **_suspect_** that somewhere in the pipeline the detections or the metric computations don't take into account the change in resolution, thus comparing detections and GT not on the same scale (therefore many localization errors).
Any idea? Have anyone else encountered this problem?
Thanks in advance.

**Comment by netanel-s** [comment]
Hi @pkulzc , thanks for your response.
What I meant is that when evaluating the pretrained SSDLite-MNv2 COCO model with larger input resolution (300x300->640x480, both fixed shape and keep AR), the results are terrible.
The AP is drastically low, and there are a lot of localization errors by means of **very** low IoU of detection with GT.
Isn't the fully convolutional architecture supposed to make it possible evaluating a model with different  input resolution (but not by a lot) and yielding fair results?

When I fine-tuned the model on the new, larger, input resolution (with all other training parameters the same), I did get a boost in performance, getting better results than of the initial evaluation. But even after 200k iterations the performance is non-marginally lower than of the original, lower, resolution, while a priori the new fine-tuned model should achieve better performance.

So from the above experiences it seems that the pretrained model both doesn't "like" to have different input resolution, and even after fine-tuning on high resolution, doesn't achieve better performance.
I'm very much baffled by this, and every clarification will be highly appreciated.
Thanks!

**Comment by pkulzc** [comment]
You can use any size for your input images, they are resized by the model automatically. 

What you have changed is the resolution after resizing, this parameters affects the feature map size and how anchors are generated. Therefore you are making the model structure completely different, all pre-trained weights are not garrarented (and not supposed) to work.

I believe when loading this checkpoint, you will see warnings in the log complaining about not able to load variables from the ckpt. Please don't ignore that since that means some of your vars are trained from scratch.

Your statement of "after fine-tuning on high resolution, doesn't achieve better performance" doesn't look fair to me as you probably didn't do too much tuning. Typically higher resolution after resizing does make the model better.

**Comment by netanel-s** [comment]
Dear @pkulzc , you're right that different input resolution changes feature map sizes, but it doesn't change the number of weights of the model in a **fully convolutional** model, and there is no such warning.

The above-mentioned fine-tuning consisted of 250k iterations, which best to my knowledge is a reasonable amount of tuning.

Therefore, I'm still baffled about the situation.

**Comment by netanel-s** [comment]
@pkulzc , thank you very much for the clarification regrading modifying learning rate.
The current learning rate scheduler is as used in the original config file:
```
learning_rate: {
        exponential_decay_learning_rate {
          initial_learning_rate: 0.004
          decay_steps: 800720
          decay_factor: 0.95
        }
```
Unfortunately, I'm not an expert in learning rate scheduling. 
Do you have a specific suggestion, or a reference which can help training on COCO?
Thank you very much again, your help is very appreciated.
